src/punx/github_handler.py

```diff
@@ -59,45 +59,45 @@
 
 def get_BasicAuth_credentials(creds_file_name = None):
     """
-    get the Github Basic Authentication credentials from a local file
-        
-    GitHub requests can use *Basic Authentication* if the 
-    credentials (username and password) are provided in the 
-    local file ``__github_creds__.txt`` which is placed
-    in the same directory as this file.  
-    The credentials file is not placed under version control 
-    since it has GitHub credentials.  
-    If found, the file is parsed for ``username password`` 
-    as shown below.  Be sure to make the file
-    readable only by the user and not others.
+    get the Github Basic Authentication token from a local file
+
+    GitHub requests use an access token provided in the local file
+    ``__github_creds__.txt`` which is placed in the same directory as this
+    file. The token is unique to a user and may be generated by visiting
+    https://github.com/settings/tokens.
+
+    The credentials file is not placed under version control since it has
+    GitHub credentials. Be sure to make the file readable only by the user and
+    not others.
+
+    Returns None if there is no credentials file.
     """
     # TODO: also look in user's .config directory, use a for loop to pick first matching path
     if creds_file_name is None:
         path = os.path.dirname(__file__)
         creds_file_name = os.path.join(path, CREDS_FILE_NAME)
     if not os.path.exists(creds_file_name):
-        return
+        return None
```

Comment:
Not necessary.  Bare `return` returns `None`.

Reply:
Because this function is capable of returning something besides None. I think it is helpful to the reader to specify None here.